.env

```
@@ -2,6 +2,12 @@ KINDEST_VERSION=v1.32.0
 GO111MODULE=on
 CGO_ENABLED=1
 REDIS_VERSION=7
+# grpcurl release bundled into the provider image (see Dockerfile). To bump,
+# update the version and both checksums here; the Dockerfile is not touched.
+# Checksums are from grpcurl_<version>_checksums.txt on the GitHub release.
+GRPCURL_VERSION=1.9.3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRPCURL_VERSION key should go before the KINDEST_VERSION key

(UnorderedKey)